src/lib/php/Dao/ClearingDao.php

```diff
@@ -957,7 +957,11 @@ public function getTotalDecisionCount($uploadId, $groupId)
     $scanJobProxy = new ScanJobProxy($GLOBALS['container']->get('dao.agent'), $uploadId);
     $scanJobProxy->createAgentStatus(array('nomos', 'monk', 'ninka', 'reportImport', 'ojo'));
     $latestAgentIds = $scanJobProxy->getLatestSuccessfulAgentIds();
-    $agentIds = "ANY(VALUES(" . implode("),(", $latestAgentIds) . "))";
```

Comment:
may be  ANY($1::int[]) is a better approach

Reply:
Done, please check.